src/app/test-record/test-record.component.spec.ts

```diff
@@ -1,40 +1,51 @@
-import {async, ComponentFixture, getTestBed, TestBed} from '@angular/core/testing';
-
-import {TestRecordComponent} from './test-record.component';
-import {INITIAL_STATE, Store} from '@ngrx/store';
-import {IAppState} from '@app/store/state/app.state';
-import {hot} from 'jasmine-marbles';
-import {ActivatedRoute, convertToParamMap} from '@angular/router';
-import {Observable, of} from 'rxjs';
+import { async, ComponentFixture, getTestBed, TestBed } from '@angular/core/testing';
```

Comment:
look at all the imports in this file. not sure need many of them to support testing this file